pkgs/applications/emulators/duckstation/default.nix

```diff
@@ -42,12 +44,11 @@ mkDerivation rec {
   buildInputs = [
     SDL2
     curl
-    gtk3
-    libevdev
     libpulseaudio
+    libXrandr
     mesa
     qtbase
-    sndio
```

Comment:
sndio is technically used by Cubeb (tier 2 support (out of 4)), I don't know how important it is, though.